lib/puppet/forge/errors.rb

```diff
@@ -102,20 +102,13 @@ def initialize(options)
     #
     # @return [String] the multiline version of the error message
     def multiline
-      if @message
-        _(<<-EOS).chomp % { uri: @uri, response: @response, message: @message }
+      message = _(<<-EOS).chomp % { uri: @uri, response: @response }
 Request to Puppet Forge failed.
   The server being queried was %{uri}
   The HTTP response we received was '%{response}'
```

Comment:
@LarissaLane should blocks like this also be broken up? I think I remember that we lose the formatting anyway as we go through the translation tooling, right?

Reply:
I considered breaking it up as we have the message 'Request to Puppet Forge failed.' as a single sentence but the others don't appear in the code base anywhere else. So I just returned it to what we had before so that we'd reuse the translation we already have.

Reply:
It would be better to have this broken out into individual sentences. We use the same translation memory across all of our products (PE, Puppet, modules, etc) and we refer to receiving HTTP responses throughout our other code bases. 
Looking at the ja.po, it looks like the indentation has been preserved. 
If you separate these out, I can fix up the translation in Transifex.